### PR TITLE
Add aadattoli to metadata.yaml

### PR DESCRIPTION
**Pull request by agitter**
@aadattoli are these the correct changes?

### CODE REVIEW

**Comment by aadattoli** [comment]
@agitter yes, they are correct. I gave the gmail email because my first day at UPenn will be tomorrow and therefore my UPenn email is not active yet. Indeed i would like to have my professional email as a contact on the paper. It should be adattoli@upenn.edu. I will try to activate it ASAP

File: content/metadata.yaml [hunk 1]
```diff
@@ -152,3 +152,18 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-03-26
+       -
```
Comment (rando2) [inline content/metadata.yaml]:
```suggestion
  -
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -152,3 +152,18 @@ authors:
...
+    github: aadattoli
+    name: Anna Ada Dattoli
+    initials: AAD
+    orcid: 0000-0003-1462-831X
+    twitter: aadattoli
+    email: annaada.dattoli@gmail.com
+    contributions:
+      - writing-original draft
```
Comment (rando2) [inline content/metadata.yaml]:
```suggestion
      - Writing- original draft 
```

**Comment by rando2** [comment]
Hi @aadattoli , I just started at UPenn last month and it took a few days to get an email address. I'm going to merge it in for now, but if you'd like to change your email later, just feel free to open a new PR and we can fix it!

**Pull request merged**